### ISSUE TITLE
Scrollbar visibilty

### ISSUE THREAD

**Issue by john-brice** [alexandrehtrb/Pororoca#101]
Vertical scrollbar only partially visible on text editors when horizontal scrollbar is visible (it is fully visible when there is no horizontal scrollbar).

Windows 10 22H2 19045.4170
Pororoca 3.2.0

**Comment by alexandrehtrb** [comment]
Yes, the vertical scrollbar is in thin mode when the horizontal scrollbar is also present.
I couldn't make a "fat" scrollbar in this case, so I opened an issue in the AvaloniaEdit project to find out how to solve this.
Currently awaiting their reply.

**Comment by john-brice** [comment]
Looks like it's just miscalculating the position of the scrollbar - as you narrow the editor pane, more of the scrollbar becomes visible.

<img width="929" alt="image" src="https://github.com/alexandrehtrb/Pororoca/assets/59295049/adedc7c7-50d1-47a1-a5d5-8ee99de7bcc3">

<img width="929" alt="image" src="https://github.com/alexandrehtrb/Pororoca/assets/59295049/29c7ceda-2b53-4bd3-bf39-263e1b7b3a81">
